pkg/parse/parse.go

```diff
@@ -576,25 +576,30 @@ func AuthConfig(creds string) (*types.DockerAuthConfig, error) {
 
 // GetBuildOutput is responsible for parsing custom build output argument i.e `build --output` flag.
 // Takes `buildOutput` as string and returns BuildOutputOption
-func GetBuildOutput(buildOutput string) (define.BuildOutputOption, error) {
+func GetBuildOutput(buildOutput, image string) (define.BuildOutputOption, error) {
 	if len(buildOutput) == 1 && buildOutput == "-" {
 		// Feature parity with buildkit, output tar to stdout
 		// Read more here: https://docs.docker.com/engine/reference/commandline/build/#custom-build-outputs
 		return define.BuildOutputOption{Path: "",
 			IsDir:    false,
 			IsStdout: true}, nil
 	}
-	if !strings.Contains(buildOutput, ",") {
-		// expect default --output <dirname>
-		return define.BuildOutputOption{Path: buildOutput,
-			IsDir:    true,
-			IsStdout: false}, nil
+
+	// if !strings.Contains(buildOutput, ",") {
```

Comment:
We don't like having commented out code in PRs, Just remove it.  We can always go back to Git to get the original code.